src/database/models/product.model.js

```diff
@@ -0,0 +1,19 @@
+const {
+  STRING,
+  NUMBER,
+} = require('sequelize');
+
+module.exports = (sequelize) => {
+  const Product = sequelize.define('Product', {
```

Comment:
the name of the table should be lower case const Product = sequelize.define('product'